groovy-android-gradle-plugin/src/test/groovy/groovyx/functional/FullCompilationSpec.groovy

```diff
@@ -126,14 +126,9 @@ class FullCompilationSpec extends AndroidFunctionalSpec {
     where:
     // test common configs that touches the different way to access the classpath
     javaVersion                     | _androidPluginVersion | gradleVersion
-    'JavaVersion.VERSION_1_7'       | '2.2.0'              | '3.0'
-    'JavaVersion.VERSION_1_7'       | '2.2.0'              | '3.1'
-    'JavaVersion.VERSION_1_6'       | '2.2.3'              | '3.2'
-    'JavaVersion.VERSION_1_7'       | '2.3.0'              | '3.3'
-    'JavaVersion.VERSION_1_7'       | '2.3.0'              | '3.4'
-    'JavaVersion.VERSION_1_7'       | '2.3.1'              | '3.5'
-    'JavaVersion.VERSION_1_7'       | '2.3.2'              | '3.5'
-    'JavaVersion.VERSION_1_7'       | '2.3.3'              | '4.2'
+    'JavaVersion.VERSION_1_7'       | '3.0.0'              | '4.2'
```

Comment:
You might want to also add Gradle version `4.1` here, which is still the default for a new project created with Android Studio 3.0

Reply:
Thanks for pointing that out. I'll add 4.1 as well.